src/Stream.sol

```diff
@@ -115,11 +116,10 @@ contract Stream is IStream, Initializable, ReentrancyGuard {
         uint256 duration = _stopTime - _startTime;
 
         if (_tokenAmount < duration) revert TokenAmountLessThanDuration();
-        if (_tokenAmount % duration != 0) revert TokenAmountNotMultipleOfDuration();
 
         remainingBalance = _tokenAmount;
         tokenAmount = _tokenAmount;
-        ratePerSecond = _tokenAmount / duration;
+        ratePerSecond = (RATE_DECIMALS * _tokenAmount) / duration;
```

Comment:
let's add documentation here that this is rounding down ratePerSecond
and explain the max value of the rounding error, and that it becomes available when the stream ends

back of the napkin:
`M (modulo) = duration`
max rounding down is `M-1`
the longer the duration, the larger the error
this error amount is the total taken off `_tokenAmount`
let's assume duration of 1 year = 31,557,600
max rounding down = 31,557,599, without decimals this would mean ~31.6 USDC is "shaved off"

because `ratePerSecond` has 6 decimals, is means for a duration of 1 year, `ratePerSecond` is rounded down by
31,557,600 / 10^6 = ~ 0.000031 USDC which is negligible
and anyway this amount is recovered at the end of the stream

Reply:
thank you, added! please take a look?